src/ctype/CTypeSchema.ts

```diff
@@ -133,7 +133,46 @@ export const CTypeWrapperModel = {
       type: 'string',
       minLength: 1,
     },
-    metamodel: {
+  },
+  required: ['schema'],
+}
+
+export const CTypeWrapperMetadata = {
+  metadata: {
+    $id: 'http://kilt-protocol.org/draft-01/ctype-metadata',
+    $schema: 'http://json-schema.org/draft-07/schema#',
+    title: {
+      type: ' string',
+    },
+    description: {
+      type: 'string',
+    },
+    properties: {
+      type: 'object',
+      properties: {},
+      patternProperties: {
+        '^.*$': {
+          type: 'object',
+          properties: {},
+        },
+      },
+    },
+  },
+  ctypeHash: { type: 'string', minLength: 1 },
+  required: ['metadata', 'ctypeHash'],
+}
+
+export const WrapperMetadata = {
```

Comment:
Is this used anywhere?

Reply:
I think it is used to check the Metadata model.